admin/admin.controller.js

```diff
@@ -86,10 +89,48 @@ const uptime = (req, res) => {
   res.send('' + uptime);
 };
 
+/**
+ * stats  Return current status for evaluator and environments.
+ *        Evaluator stats in response:
+ *          - HealthCheck (Version, sdk and sdkVersion)
+ *          - UpTime
+ *        Stats for each environment:
+ *          - readiness
+ *          - splits
+ *          - segments
+ *          - impressionsMode
+ *          - last Evaluation
+ *
+ * 200 - Everything is OK. Evaluator returns stats.
+ * 500 - Something is wrong or not ready yet. Not able to get stats.
+ * @param {*} req
+ * @param {*} res
+ */
+const stats = (req, res) => {
+  console.log('Running stats.');
```

Comment:
is this needed?

Reply:
each endpoint on admin controller logs when is called, if you think that is not necessary I could remove it, I added it just to be consistent

Reply:
np